packages/utils/telemetry-utils/src/logger.ts

```diff
@@ -31,6 +31,8 @@ export interface ITelemetryLoggerPropertyBags{
  * Creates sub-logger that appends properties to all events
  */
 export abstract class TelemetryLogger implements ITelemetryLogger {
+    public supportsTags: true = true;
```

Comment:
NOT CORRECT -- @ksbrar is working on properly adding support for `supportsTags` now.  This will be abstract and `MultsinkLogger` and `ChildLogger` will add proper logic for handling this.